package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "fantasy-laws",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "test": "make lint test"
+  },
+  "dependencies": {
+    "fantasy-land": "3.1.x",
+    "jsverify": "0.8.x",
+    "sanctuary-type-classes": "3.0.x"
```

Comment:
We must bump this to `"3.1.x"` once v3.1.0 has been published in order to have access to `Z.contramap`.